pkgs/applications/blockchains/parity/parity.nix

```diff
@@ -26,10 +26,7 @@ rustPlatform.buildRustPackage rec {
     inherit sha256;
   };
 
-  buildInputs = [
-    pkgconfig cmake perl
-    systemd.lib systemd.dev openssl openssl.dev
-  ];
+  buildInputs = [ cmake openssl perl pkgconfig systemd ];
```

Comment:
Are we sure we need `cmake` and `pkgconfig` as `buildInputs`? Those are typically only needed during the build so they're `nativeBuildInputs`.

Reply:
Also where is perl needed https://github.com/paritytech/parity-ethereum/search?q=perl&type=Code ?